### DISCUSSION TITLE
Possible to build browser like apps?

### DISCUSSION THREAD

**Discussion by wenerme** [wailsapp/wails#1452] [Q&A]
### Is your feature request related to a problem? Please describe.

Build a browser like app, like BrowserView or webview in electron

### Describe the solution you'd like

overlay webview like BrowserView in electron

### Describe alternatives you've considered

iframe ?

**Comment by leaanthony** [comment]
This project is for building such apps 👍

**Reply by wenerme** [reply]
Browser like I mean like brave is can be build on electron. like https://github.com/wexond/browser-base
How wails manage multi window ? I don't find any reference.

**Reply by leaanthony** [reply]
No multi window currently but planned after v2 release.

**Reply by wenerme** [reply]
Thanks, good to know that.

**Reply by wenerme** [reply]
xref tauri same problem here https://github.com/tauri-apps/tauri/issues/2709
